galasa-simplatform-application/src/main/java/dev/galasa/simplatform/listener/ListenerManager.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Licensed Materials - Property of IBM
+ * 
+ * (c) Copyright IBM Corp. 2019.
+ */
+package dev.galasa.simplatform.listener;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.io.PrintStream;
+import java.net.Socket;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.StringTokenizer;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/*
+ * 	ListenerManager acts as a head listener for all current
+ * 	and future listeners. It takes the request from the sender
+ * 	and refers to the correct listener.
+ * 
+ * 	This solves the issue of having multiple listeners 
+ * 	waiting for requests across multiple ports.
+ * 
+ *	Written by @Lewis James on 17/12/2019
+ * 
+ */
+
+public class ListenerManager implements IListener{
```

Comment:
Not sure if this super class needs to implement IListener.  Can we roll IListener into this super class